changelog/5214.logging.rst

```diff
@@ -0,0 +1 @@
+Improve the default logging format and show module name instead of filename.
```

Comment:
Because this is changing existing functionality, it should be called `5214.feature.rst`. 👍